src/components/crossDevice/MobileFlow.js

```diff
@@ -33,15 +36,22 @@ class MobileFlow extends Component {
 
   onClientSuccess = () => {
     this.props.actions.setClientSuccess(true)
+    this.props.actions.mobileConnected(false)
   }
 
   onDisconnectPing = (data) => {
     this.sendMessage('disconnect pong', data.roomId)
+    this.props.actions.mobileConnected(false)
```

Comment:
This introduces quite a subtle bug - open two mobile clients then close one of them and the desktop goes to "Check your mobile". We need to be more systematic about the set of states our app can be in - happy to talk it through at the whiteboard

Reply:
This has been fixed